src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/twig.html.twig

```diff
@@ -0,0 +1,85 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% set time = collector.templatecount|length ? '%0.0f ms'|format(collector.time) : 'n/a' %}
+    {% set icon %}
+        <img height="28" alt="Twig" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABUAAAAcCAYAAACOGPReAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAbElEQVRIx2NgGAXUBowMDAwMaWlp/6ll4KxZsxhZYJy0tDRqGMjAwMDAwEQL77OgCxSXlJBsSG9PDwqfJi6lj/fRvTJ4XYocUTBXE4q8oRtRRBnKwsw8RFw6fA0lKkd1dnYOIpfCCthRMIIAAI0IFu9Hxh7ZAAAAAElFTkSuQmCC" />
+        <span>{{ time }}</span>
```

Comment:
you should add ``class="sf-toolbar-status"`` so that the time gets hidden on smaller screens (smartphones for instance)